src/applications/_mock-form/components/GroupCheckboxWidget.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { VaCheckboxGroup } from 'web-components/react-bindings';
```

Comment:
I know this is how the react bindings are imported in some places, but can we use the true path instead of [the aliased one](https://github.com/department-of-veterans-affairs/vets-website/blob/ac60e8f296360efbc597516f5947139b25b4210f/.babelrc#L63)? I'd like to eventually get rid of that.

There's also an [example in the documentation](https://design.va.gov/documentation/developers#react-applications):

> ```jsx
> import { VaExampleComponent } from "@department-of-veterans-affairs/component-library/dist/react-bindings";
> ```